modules/Administration/controller.php

```diff
@@ -166,4 +166,9 @@ public function action_callRebuildSprites()
             $GLOBALS['log']->error($mod_strings['LBL_SPRITES_NOT_SUPPORTED']);
         }
     }
+
+    public function action_diagnosticquickviewphpinfo() {
+        phpinfo();
```

Comment:
Hi @pgorod, could we add a check here to make sure the user is an admin user, please?

Reply:
Ok, will do. I will also add it to the other actions in this controller that are missing it.